common/store.go

```diff
@@ -248,7 +251,10 @@ func createSchemaMysql(dbStore *DbStore, force bool) error {
 
 	for i := range entities {
 		entity := entities[i]
-		dbStore.Db.CreateTable(entity)
+		db := dbStore.Db.CreateTable(entity)
```

Comment:
Usually DBs do not support rollback for DDL operations... So we are on our own here.